dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Trn.cshtml

```diff
@@ -12,13 +12,6 @@
     <govuk-panel-body>
         <vc:client-scoped-partial view-name="_TrnLookup.StartBookend.{0}.Content" />
 
-        <form action="@Model.HandoverUrl" method="@Model.HandoverMethod" asp-antiforgery="false">
-            @foreach (var p in Model.HandoverParameters!)
-            {
-                <input type="hidden" name="@(p.Key)" value="@(p.Value)" />
-            }
-
-            <govuk-button type="submit" class="app-button--inverse govuk-!-margin-bottom-0">Continue</govuk-button>
-        </form>
+        <govuk-button-link href="@Model.NextPage" class="app-button--inverse govuk-!-margin-bottom-0">Continue</govuk-button-link>
```

Comment:
The form here was a hangover from when we used to POST to Find a lost TRN at this point in the journey. Now that's gone, a link is sufficient.